.travis.yml

```diff
@@ -69,6 +71,36 @@ matrix:
      script:
       - RUSTFLAGS="-D warnings" cargo build --release --verbose
       - RUSTFLAGS="-D warnings" cargo test --verbose
+   - language: android
+     dist: trusty
+     android:
+       components:
+        - build-tools-26.0.1
+        # Minimum API level supported
+        - android-21
+     install:
+      # Install rust manually
+      - curl https://build.travis-ci.org/files/rustup-init.sh -sSf | sh -s -- -y --default-toolchain stable
+      - export PATH=$HOME/.cargo/bin:$PATH
+      - rustup default stable
+      - rustup target add aarch64-linux-android arm-linux-androideabi armv7-linux-androideabi i686-linux-android
+      # Install android ndk
```

Comment:
Can we use the `sdkmanager` command for this? https://github.com/android/ndk-samples/blob/master/.travis.yml#L18

Reply:
Here I am reusing how other rust project building with ndk (e.g. libc, ring).  I will take a look again.

Reply:
Now we use `sdkmanager` for `cmake`. I tested installing `ndk-bundle` with `sdkmanager` but it fails... I think it's because `ndk-bundle` coming with this `sdkmanager` is NDK r20 and we are not still compatible with this. There is toolchain script change somewhere between 13b and 20, breaking backward compatibility with old NDK. r13b was chosen based on current practice. If we want to be compatible with NDK 20 then we need a separate work.